microsoft-edge/webview2/how-to/prerelease-testing.md

```diff
@@ -0,0 +1,189 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 03/01/2024
+---
+# Prerelease testing using preview channels
+
+_Prerelease testing_ means testing your WebView2 app by using the preview channels of Microsoft Edge to find issues that will affect your particular WebView2 app, before any changes ship in the Stable Evergreen Runtime.  Prerelease testing helps your app catch bugs in prerelease channels of WebView2, before the WebView2 changes ship in a stable release and degrade the app experience for end users.  In prerelease testing, you validate your app against early builds of WebView2, to ensure that your app will work with the upcoming changes.
+
+To validate your app against early builds of WebView2, do automated testing with the preview channels of Microsoft Edge.  A preview channel contains a WebView2 preview runtime.
+
+The preview (or "insider") channels of Microsoft Edge consist of:
+* Edge Canary - releases daily.
+* Edge Dev - releases weekly.
+* Edge Beta - releases every 4 weeks.
+
+See also: 
+* [Overview of the Microsoft Edge channels](/deployedge/microsoft-edge-channels)
+* [Become a Microsoft Edge Insider](https://www.microsoft.com/edge/download/insider) - download Edge Canary, Edge Dev, or Edge Beta
+* [Self-host by deploying preview channels](./self-hosting.md)
+
+
+<!-- ====================================================================== -->
+## Importance of prerelease testing
+
+Evergreen WebView2 is based on the evergreen Chromium platform, which receives monthly major updates.  Regressions in WebView2 apps tend to be application-specific, rather than affecting all WebView2 apps.  WebView2 is tested in a variety of general scenarios, but it's possible that some of the specific scenarios of your app are not covered.
+
+To catch app-specific regressions before WebView2 changes ship in Edge Stable, test your WebView2 app against the prerelease runtime that's shipped with the prerelease channels of Microsoft Edge (Edge Canary, Edge Dev, or Edge Beta).
+
+
+<!-- ====================================================================== -->
+## Start by doing automated testing against Edge Canary
+
+Start with automated testing against the Microsoft Edge **Canary** channel, which ships daily.  This enables you to catch issues as early as possible. The Microsoft Edge **Dev** channel ships weekly, and is also a good option.
+
+Do the following steps:
+
+1. Set the preview channel for your app.  See [Set the preview channel by using ChannelSearchKind](#set-the-preview-channel-by-using-channelsearchkind), below.
+
+1. Deploy the non-stable channels.  See [How to deploy non-stable channels](#how-to-deploy-non-stable-channels), below.
```

Comment:
I agree on this point. Thanks for pointing it out in the review. I think everyone who has touched this document has a different name for it. I will change everything to "Preview", because calling them pre-release will confuse people with the pre-release SDK. Non-stable doesn't quite convey our intentions.